jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/WebAppClassLoader.java

```diff
@@ -637,4 +646,34 @@ public boolean isServerClass(Class<?> clazz)
     {
         return _context.isServerClass(clazz);
     }
+
+    private static class UniqueURIPredicate implements Predicate<URI>
+    {
+        private final List<URI> uris = new ArrayList<>();
```

Comment:
Can't this be a `HashSet` instead of an `ArrayList`?

Reply:
Sure, why not.